kmod/src/inode.c

```diff
@@ -1566,12 +1565,15 @@ int scoutfs_drop_inode(struct inode *inode)
  * This only scans orphans for this node.  This will need to be covered by
  * the rest of node zone cleanup.
  */
-int scoutfs_scan_orphans(struct super_block *sb)
+int scoutfs_scan_orphans(struct super_block *sb,
+			 struct scoutfs_bloom_block *bb)
 {
 	struct scoutfs_sb_info *sbi = SCOUTFS_SB(sb);
 	struct scoutfs_lock *lock = sbi->rid_lock;
 	struct scoutfs_key key;
 	struct scoutfs_key last;
+	unsigned int deleted = 0;
+	u64 seq;
```

Comment:
This is pulled from the item value and needs to be __le64, maybe also call it last_seq to match the orphan creation side?